ui/raidboss/data/07-dt/raid/r2n.txt

```diff
@@ -1,8 +1,77 @@
+146.1 "Love Me Tender" Ability { id: "9174", source: "Honey B. Lovely" }
+152.2 "Heart-struck" duration 3.2
+160.7 "Blow Kiss" Ability { id: "9173", source: "Honey B. Lovely" }
+171.5 "Honey B. Finale" Ability { id: "917B", source: "Honey B. Lovely" }
+# End first hearts phase
+
+180.6 "Honeyed Breeze" Ability { id: "9167", source: "Honey B. Lovely" }
+186.8 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+192.1 "Splash of Venom" Ability { id: "9169", source: "Honey B. Lovely" }
+202.6 "Tempting Twist/Honey Beeline" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
+212.9 "Splash of Venom/Drop of Venom" Ability { id: ["916F", "9170"], source: "Honey B. Lovely" }
+219.9 "Drop of Venom" Ability { id: "916A", source: "Honey B. Lovely" }
+229.4 "Tempting Twist/Honey Beeline" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
+239.7 "Splash of Venom/Drop of Venom" Ability { id: ["916F", "9170"], source: "Honey B. Lovely" }
+251.8 "Alarm Pheromones" Ability { id: "917D", source: "Honey B. Lovely" }
+251.8 "Blinding Love" duration 34.3
+293.1 "Tempting Twist/Honey Beeline" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
+
+299.7 label "loop"
+299.7 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+310.4 "Honey B. Live" Ability { id: "9A7E", source: "Honey B. Lovely" }
+
+# Second hearts phase
+315.2 "Love Me Tender" Ability { id: "9174", source: "Honey B. Lovely" }
+323.3 "Heartsick" Ability { id: "9B8D", source: "Honey B. Lovely" }
+323.3 "Fracture x6" duration 17
+339.3 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+345.5 "Loveseeker" Ability { id: "9AC1", source: "Honey B. Lovely" }
+345.5 "Heart-struck" duration 42.1
+350.6 "Love Me Tender" Ability { id: "9174", source: "Honey B. Lovely" }
+376.9 "Blow Kiss" Ability { id: "9173", source: "Honey B. Lovely" }
+387.6 "Honey B. Finale" Ability { id: "917B", source: "Honey B. Lovely" }
+# End second hearts phase
+
+396.7 "Honeyed Breeze" Ability { id: "9167", source: "Honey B. Lovely" }
+402.9 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+408.1 "Alarm Pheromones" Ability { id: "917D", source: "Honey B. Lovely" }
+408.1 "Blinding Love" duration 28.3
+413.1 "Drop of Venom" Ability { id: "916A", source: "Honey B. Lovely" }
+436.2 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+445.4 "Tempting Twist/Honey Beeline" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
+455.7 "Splash of Venom/Drop of Venom" Ability { id: ["916F", "9170"], source: "Honey B. Lovely" }
+456.7 "Splash of Venom" Ability { id: "9169", source: "Honey B. Lovely" }
+463.8 "--middle--" Ability { id: "9163", source: "Honey B. Lovely" }
+473.6 "Tempting Twist/Honey Beeline" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
+483.9 "Splash of Venom/Drop of Venom" Ability { id: ["916F", "9170"], source: "Honey B. Lovely" }
```

Comment:
```suggestion
473.6 "Honey Beeline/Tempting Twist" Ability { id: ["9B39", "9B3A", "9B3B", "9B3C"], source: "Honey B. Lovely" }
483.9 "Drop of Venom/Splash of Venom" Ability { id: ["916F", "9170"], source: "Honey B. Lovely" }
```
These are always reversed from the first occurrence in the loop, right? If it's always a fixed opposite, I think we usually try to reverse the order of the abilities in each line like so, just to indicate it a little.

Reply:
I didn't test enough pulls to confirm that they're always reversed, so I didn't reverse them here. If we have reasonable confirmation (10 pulls maybe?) that this is the case, I'm fine with swapping them.